extensions/database-documentation/package.json

```diff
@@ -0,0 +1,65 @@
+{
+	"name": "database-documentation",
+	"displayName": "%displayName%",
+	"description": "%description%",
+	"publisher": "Microsoft",
+	"version": "0.0.1",
+	"license": "https://raw.githubusercontent.com/Microsoft/azuredatastudio/main/LICENSE.txt",
+	"engines": {
+		"vscode": "^1.25.0",
+		"azdata": ">=1.44.0"
+	},
+	"categories": [
+		"Other"
+	],
+	"activationEvents": [
+		"*"
```

Comment:
Please make an issue to track finding a better activation event for this - we should have as few * activations as possible.